lib/emergency_banner/display.rb

```diff
@@ -0,0 +1,34 @@
+module EmergencyBanner
+  class Display
+    def enabled?
+      content && campaign_class && heading
+    end
+
+    def campaign_class
+      content[:campaign_class] if content[:campaign_class].present?
+    end
+
+    def heading
+      content[:heading] if content[:heading].present?
+    end
+
+    def short_description
+      content[:short_description] if content[:short_description].present?
+    end
+
+    def link
+      content[:link] if content[:link].present?
+    end
+
+  private
+
+    def content
+      @data ||= connect.hgetall("emergency_banner")
```

Comment:
@boffbowsh I'm not sure if we need this. `campaign_class` and `heading` must both be defined for the banner to show. So if one fails, and the other succeeds, the user still won't see the banner.

It's also possible and ok for the `emergency_banner` key to not exist (return nil) in Redis. The `emergency_banner` key will be nil in an environment until the first time it is set from the rake task. As the banner is only going to be "up" in rare occasions I think it's ok not to notify Errbit that the emergency banner content could not be found.